### PR TITLE
Tests navigation through first Related Content link - forward and back

### PR DESCRIPTION
**Pull request by LilyL0u**
**Overall change:** 
If the related content section has at least one item in its list, click the first link and check that the new page loaded has the expected URL. Then navigates backwards one using cy.go, like using the back button, and checks that the URL is back to the MAP page the user originated from.
**Code changes:**

-Requests the json and gets the array length of the related content story promo list.
-If the array contains at least one item (>0), 
-Get the assetURI on the page and store in variable for comparison later
-Click on the link in the first list item
-Get the URL of the newly loaded page (cy.get)
-Assert the new URL includes the assetURI from the link clicked on the first page (substring using should 'include', as things like candy override are added to the URL)
-Go back to the previous page using cy.go(-1)
-Assert the URL includes the original URL from the first page the user was on.
---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**

- [x] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 

This passes on local and test, although on local OTHER TESTS fail

### CODE REVIEW

**Review by karinathomasbbc — APPROVED** [review]

File: cypress/integration/pages/mediaAssetPage/tests.js [hunk 1]
```diff
@@ -66,12 +66,36 @@ export const testsThatFollowSmokeTestConfig = ({
...
+          if (arrayLength > 0) {
+            const assetURI =
+              body.relatedContent.groups[0].promos[0].locators.assetUri;
+            cy.get('li[class^="StoryPromoLi"]')
+              .first()
+              .click();
+
+            cy.url().should('include', assetURI);
+
+            cy.go(-1);
```
Comment (karinathomasbbc) [inline cypress/integration/pages/mediaAssetPage/tests.js:90]: Could you maybe use `cy.go('back')` - just for improved readability

File: cypress/integration/pages/mediaAssetPage/tests.js [hunk 2]
```diff
@@ -66,12 +66,36 @@ export const testsThatFollowSmokeTestConfig = ({
...
     });
+
+    it('should navigate to first related content link, and back again', () => {
+      cy.request(`${config[service].pageTypes[pageType].path}.json`).then(
+        ({ body }) => {
+          const arrayLength = body.relatedContent.groups.length;
+
+          if (arrayLength > 0) {
+            const assetURI =
+              body.relatedContent.groups[0].promos[0].locators.assetUri;
```
Comment (ryanmccombe) [inline cypress/integration/pages/mediaAssetPage/tests.js:83]: This seems prone to throw exceptions, unless we know that `promos[0].locators.assetUri` is always defined as long as `groups` has a length?

**Review by ryanmccombe — APPROVED** [review]
Looks good - as long as https://github.com/bbc/simorgh/pull/5042/files#r363382980 is true :)

File: cypress/integration/pages/mediaAssetPage/tests.js (hunk 2 above)
Comment (LilyL0u) [reply cypress/integration/pages/mediaAssetPage/tests.js:83]: On all the example pages on local and test this seemed to be fine (always was defined if there was content in groups, if no content in groups it skips the test), but I can have a look at a few more pages to check this.

**Pull request merged**